tests/lib/test_openstack_floating_ip.py

```diff
@@ -299,3 +295,71 @@ def test_search_fips_down_before(self, mock_datetime):
         )
 
         self.assertEqual(result, [self.mock_fip_list[0]])
+
+    def test_find_non_existent_fips(self):
+        """
+        Tests calling find_non_existent_fips
+        """
+        # pylint:disable=too-few-public-methods,invalid-name,redefined-builtin
+        class ObjectMock:
```

Comment:
So `redefined-builtin` is something we don't want to disable, as it can break in unexpected ways for future devs as you're effectively shadowing builtin objects.

In this case I would simply create a stub dataclass like so:
```
@dataclass
class _StubbedReturn:
    id: str
    project_id: str

def test_foo():
    instance = _StubbedReturn("bar", "baz")
    ...
```

And let the dataclass handle it for you, since we're not actually calling these so a mock is a touch overkill

Reply:
The redefined-builtin was actually due to id which must be that name since it is from the openstack API, but using a dataclass fixed it anyway since it was only a problem in the constructor.